File: backend/SHBox.API/Models/Photo.cs
namespace SHBox.API.Models;

public class Photo
{
    public int Id { get; set; }

    public string FileName { get; set; } = "";

    public string Sender { get; set; } = "";

    public string Caption { get; set; } = "";

    public DateTime Timestamp { get; set; } = DateTime.UtcNow;
}

File: frontend/mobile_app/lib/models/photo.dart
class Photo {

  final int id;
  final String fileName;
  final String sender;
  final String caption;
  final DateTime timestamp;


  Photo({
    required this.id,
    required this.fileName,
    required this.sender,
    required this.caption,
    required this.timestamp,
  });


  factory Photo.fromJson(
      Map<String,dynamic> json)
  {
    return Photo(

      id: json['id'],

      fileName:
      json['fileName'],

      sender:
      json['sender'],

      caption:
      json['caption'],

      timestamp:
      DateTime.parse(
          json['timestamp']
      ),

    );
  }
}